core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/CollectionUtil.java

```diff
@@ -46,4 +47,36 @@ private CollectionUtil() {
 
         return treeSet;
     }
+
+    public static boolean containsAny(final Collection collection, final Object[] objects) {
```

Comment:
Those 2 methods takes collection, collection. We need to ones that works with an array. I could do Arrays.asList(array) and then use those methods but it seemed wasteful to create an extra object. What do you think?